### ISSUE TITLE
Issue with Offhand Crystal (delay in breaking)

### ISSUE THREAD

**Issue by lollipop-arch** [TrvsF/wurstplus-two#57]
For crystal pvp I use Crystal aura and offhand crystal config. When crystal aura is on and crystal is in offhand, Wurstplustwo places the crystal but takes 2 seconds to break it. With offhand crystal mode off and using crystal aura only, it takes 13 seconds for a stack of crystals to be destroyed. There is some delay when using the offhand feature.
The debug looks something like this:
place
place
place
place
place
place
break
place
place
etc
Crystals cannot be placed in the same position until the previous one is broken. I tried playing around with the settings but nothing helped. I use W+2 with Future but only rely on W+2 for crystal stuff. I also experienced the problem with only W+2 running. Can issue be looked into?

**Comment by CynicalMuffin** [comment]
ur ping and config matters
if ur ping is shit then close the issue rn
if ur config is shit do the same

**Comment by TrvsF** [comment]
tldr, person above is probably right
